manifests/params.pp

```diff
@@ -74,12 +74,23 @@
       $site_packages = "/usr/local/lib/python${version}/dist-packages"
     }
     redhat: {
+      case $::operatingsystemmajrelease {
+        '6': {
+          $version       = '2.6'
+          $pip           = 'python-pip'
+        }
+        '7': {
+          $version       = '2.7'
+          $pip           = 'python2-pip'
+        }
+        default: {
+          fail("redhat version \"${::operatingsystemmajrelease}\" not supported")
```

Comment:
I'm not quiet familiar with RedHat either. I think you are right and we can expect older versions to still support `pip`, but I don't know for sure. Let me try to fire up a CentOS 5 instance on EC2 and get back to you.